ReactiveUIKit/UICollectionView.swift

```diff
@@ -72,19 +72,21 @@ public class RKCollectionViewDataSource<C: ObservableCollectionType where C.Coll
     
     collectionView.dataSource = self
     collectionView.reloadData()
-
+    
     observableCollection.skip(1).observe(on: ImmediateOnMainExecutionContext) { [weak self] event in
```

Comment:
You can just change `ImmediateOnMainExecutionContext` to `MainExecutionContext` and it will do `dispatch_async(dispatch_get_main_queue()...` :)